bitnami/cert-manager/README.md

```diff
@@ -0,0 +1,336 @@
+| `cainjector.serviceAccount.create`                 | Specifies whether a ServiceAccount should be created                                                 | `true`               |
+| `cainjector.serviceAccount.name`                   | The name of the ServiceAccount to use.                                                               | `""`                 |
+| `cainjector.serviceAccount.annotations`            | Additional custom annotations for the ServiceAccount                                                 | `undefined`          |
+
+
+### Metrics Parameters
+
+| Name                                      | Description                                                                                      | Value       |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------ | ----------- |
+| `metrics.enabled`                         | Start metrics                                                                                    | `true`      |
+| `metrics.podAnnotations`                  | Annotations for Cert Manager exporter pods                                                       | `{}`        |
+| `metrics.serviceMonitor.enabled`          | Create ServiceMonitor resource(s) for scraping metrics using PrometheusOperator                  | `false`     |
+| `metrics.serviceMonitor.namespace`        | The namespace in which the ServiceMonitor will be created                                        | `nil`       |
+| `metrics.serviceMonitor.interval`         | The interval at which metrics should be scraped                                                  | `60s`       |
+| `metrics.serviceMonitor.path`             | The path which the ServiceMonitor will monitor                                                   | `/metrics`  |
+| `metrics.serviceMonitor.scrapeTimeout`    | The timeout after which the scrape is ended                                                      | `30s`       |
+| `metrics.serviceMonitor.additionalLabels` | Additional labels that can be used so ServiceMonitor resource(s) can be discovered by Prometheus | `undefined` |
+
+
+### Other Parameters
+
+| Name          | Description                                        | Value  |
+| ------------- | -------------------------------------------------- | ------ |
+| `rbac.create` | Specifies whether RBAC resources should be created | `true` |
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install my-release bitnami/cert-manager \
+  --set installCRDs=true
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install my-release -f values.yaml bitnami/cert-manager
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Configuration and installation details
```

Comment:
We should include a section explaining how to create different Issuers (e.g. SelfSigned & ACMESolver)

Reply:
I am working on it. If we decided finally to merge the change, I will create another PR adding those new sections.